optimum/exporters/onnx/__main__.py

```diff
@@ -128,7 +128,8 @@ def main():
         maybe_save_preprocessors(args.model, args.output.parent)
 
     if task == "stable-diffusion" or (
-        task.startswith(("causal-lm", "seq2seq-lm", "speech2seq-lm", "vision2seq-lm")) and not args.monolith
+        task.startswith(("causal-lm", "seq2seq-lm", "speech2seq-lm", "vision2seq-lm", "default-with-past"))
```

Comment:
Adding `default-with-past` is just a hack, because `args.monolith` is `False` by default even if `task="default"`. Will do an other PR to clean.